packages/gestalt/src/Contents.js

```diff
@@ -474,6 +470,11 @@ export default class Contents extends React.Component<Props, State> {
     });
   };
 
+  // eslint-disable-next-line camelcase
+  UNSAFE_componentWillReceiveProps(nextProps: Props) {
```

Comment:
@janephilipps Could we instead use `allow` for this rule in our eslint config? https://eslint.org/docs/rules/camelcase#allow
```
/*eslint camelcase: ["error", {allow: ["UNSAFE_componentWillMount"]}]*/

function UNSAFE_componentWillMount() {
    // ...
}
```